src/applications/vaos/services/appointment/index.js

```diff
@@ -299,6 +299,26 @@ export function isValidPastAppointment(appt) {
   );
 }
 
+/**
+ * Checks to see if the appointment is either an appointment that does
+ * not have one of the excluded statuses for past appointments or an
+ * Express Care request that is either resolved or old enough to show
+ *
+ * @param {Object} appt A FHIR appointment resource
+ * @returns Whether or not the appt should be shown
+ */
+export function isValidPastAppointmentOrExpressCare(appt) {
+  return (
+    (appt.vaos.isExpressCare && appt.status === APPOINTMENT_STATUS.fulfilled) ||
+    (appt.vaos.isExpressCare &&
+      appt.status !== APPOINTMENT_STATUS.fulfilled &&
+      moment(appt.created).isBefore(moment().subtract(2, 'days'))) ||
+    (appt.vaos.appointmentType === APPOINTMENT_TYPES.vaAppointment &&
+      !PAST_APPOINTMENTS_HIDDEN_SET.has(appt.description)) ||
+    appt.vaos.appointmentType === APPOINTMENT_TYPES.ccAppointment
```

Comment:
I've seen this monster before and was wondering about how to refactor it but I'm not so sure some of the refactoring techniques actually make this easier to reason about.

Reply:
I think it could use some more comments, probably, to call out the 4 distinct pieces.

Reply:
Added some comments